spec/views/users/user_index_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'User index', type: :feature do
+  before :each do
+    @user1 = User.create(name: 'Ali', photo: 'https://picsum.photos/300/200', bio: 'Software Engineer from Pakistan',
+                         posts_counter: 12)
+    @user2 = User.create(name: 'Hassan', photo: 'https://picsum.photos/300/200', bio: 'Software Engineer from Morocco',
+                         posts_counter: 2)
+    visit users_path
+  end
+
+  it 'Show user name' do
+    expect(page).to have_content(@user1.name)
+    expect(page).to have_content(@user2.name)
+  end
```

Comment:
**OPTIONAL** - Good job done so far, a more efficient way to do this would be, can be applied in all similar scenarios, please fix this:
``` 
@users = User.all
     @user.each |user| do
          expect(page).to have_content(user.name)
        end
```